workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/camunda/WorkflowEventToCamundaEvent.java

```diff
@@ -304,7 +304,14 @@ private <T> void formReplyToMessage(RealTimeEvent<T> event, Map<String, Object>
   @SuppressWarnings({"unchecked", "rawtypes"})
   private void requestReceivedToRequest(RequestReceivedEvent eventSource, Map<String, Object> processVariables) {
     String eventName = requestReceivedWorkflowEvent(eventSource.getWorkflowId());
-    Map<String, Object> args = new HashMap<>(eventSource.getArguments());
+    Map<String, Object> args;
+
+    if (eventSource.getArguments() == null) {
```

Comment:
Suggest it to be like

`Map<String, Object> args = new HashMap<>();

    if (eventSource.getArguments() != null) {
      args.putAll(eventSource.getArguments());
    }`

Reply:
eventSource.getArguments() is immutable, that's why I'm cloning it in the "else" .